docs/provider-pkcs11.7.md

```diff
@@ -258,6 +258,34 @@ Examples:
 ```PKCS11_PROVIDER_DEBUG=file:/dev/stderr,level:2```
 
 
+USE IN OLDER APPLICATIONS (URIs in PEM files)
+=============================================
+
+It is strongly suggested to update applications to use the new
+OSSL_STORE API provided by OpenSSL 3.0 which accepts URIs to
+transparenly load keys from either files or any other supported
+mechanism including pkcs11 URIs.
+
+However, for those applications that cannot yet be changed there is
```

Comment:
not sure about the english rules for the commas, but I feel like there should be one more separating the sub sentence:
```suggestion
However, for those applications that cannot yet be changed, there is
```